examples/step-22/doc/intro.dox

```diff
@@ -767,7 +767,8 @@ comparison of their results.
 Apart from this, many other algorithms have been tested and improved during
 the creation of this program. For example, in building the sparsity pattern,
 we originally used a BlockCompressedSparsityPattern object that added one
-element at a time; however, its data structures are poorly adapted for the
+element at a time (this no longer exists in deal.II); however, its data
+structures are poorly adapted for the
```

Comment:
I think it would be clearer to write "this class no longer exists" instead of "this no longer exists".

Not particularly important, but it would make more sense to refer to `BlockCompressedSparsityPattern` in the past tense here (i.e., write "its data structures were poorly adapted").